environ/funcs.go

```diff
@@ -138,6 +139,15 @@ func numbers(start, end int) data.Strings {
 	return s
 }
 
+// steps returns a live of strings of the numbers start with size of len
```

Comment:
What's a "live" of strings? I don't really understand this comment at all and I can't infer from the code what it's supposed to say.

Reply:
That was a misspell. steps act like numbers return a string from start with the length of len.